content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md

```diff
@@ -0,0 +1,193 @@
+
+1. Ensure you are in the correct project scope.
+
+
+2. Navigate to the left **Main Menu** and click on **Clusters**
+
+
+3. Click on **Add New Cluster**
+
+
+4. You will receive a prompt asking you if you want to deploy a new cluster or import an existing cluster. Click on **Deploy New Cluster**
+
+
+5. Select **AWS** and click on **Start AWS Configuration**
+
+
+6. Populate the wizard page with the following information: name, description, tags and AWS account. Tags on a cluster are propagated to the VMs deployed on the cloud/data center environments. Click on **Next** after you have filled out all the required information.
+
+
+7. Select a cluster profile. Click on **Next**.
+
+
+8. Review and customize pack parameters, as desired. By default, parameters for all packs are set with values, defined in the cluster profile.
+
+
+9. Provide the AWS cloud account and placement information.
+
+    |**Parameter**| **Description**|
+    |-------------|---------------|
+    |**Cloud Account** | Select the desired cloud account. AWS cloud accounts with AWS credentials need to be pre-configured in project settings.|
+    |**Region** | Choose the preferred AWS region where you would like the clusters to be provisioned.|
+    |**SSH Key Pair Name** | Choose the desired SSH Key pair. SSH key pairs need to be pre-configured on AWS for the desired regions. The selected key is inserted into the VMs provisioned.|
+    |**Static Placement** | By default, Palette uses dynamic placement, wherein a new VPC with a public and private subnet is created to place cluster resources for every cluster. <br /> These resources are fully managed by Palette and deleted, when the corresponding cluster is deleted. Turn on the **Static Placement** option if it's desired to place resources into preexisting VPCs and subnets.<br /> If the user is making the selection of **Static Placement** of resources, the following placement information needs to be provided:
+    ||**Virtual Network**: Select the virtual network from dropdown menu.
+    ||**Control plane Subnet**: Select the control plane network from the dropdown menu.
+    ||**Worker Network**: Select the worker network from the dropdown menu. |
+    
+
+10. Make the choice of updating the worker pool in parallel, if required. Click on **Next**.
+
+
+11. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](/clusters/public-cloud/aws/architecture#spotinstances), disk size, and the number of nodes.
+
+<InfoBox>
+
+You can add new worker pools if you need to customize certain worker nodes to run specialized workloads. As an example, the default worker pool may be configured with the m3.large instance types for general-purpose workloads, and another worker pool with instance type g2.2xlarge can be configured to run GPU workloads.
```

Comment:
For example, the default worker pool may be configured with the m3.large instance types for general-purpose workloads, and another worker pool with instance type g2.2xlarge can be configured to run GPU workloads.

i guess this is in passive voice.